kolibri/utils/version.py

```diff
@@ -103,6 +103,8 @@
 import re
 import subprocess
 
+import semver
```

Comment:
This was an error. I should either import this inside the functions that use it, or define them external to this file.